src/exchange/wizard-hosted-creation/summary/manual/manual.html

```diff
@@ -8,7 +8,7 @@
         <div data-wizard-step>
             <div class="row">
                 <div class="col-md-2 oui-modal__body pt-0 text-center">
-                    <i class="oui-icon oui-icon-warning_circle oui-icon_bicolor oui-icon-dialog" aria-hidden="true"></i>
+                    <i class="oui-icon oui-icon-warning_circle oui-icon_bicolor oui-icon-dialog oui-icon-dialog-relatively-centered" aria-hidden="true"></i>
```

Comment:
Can't find those classes from http://master.ui-kit.ovh/#!/ovh-ui-kit/icons.
Are they correct?

Reply:
I've seen them there: https://ovh-ux.github.io/ovh-ui-kit-bs/

Exception from `oui-icon-dialog-relatively-centered` that I had to created myself, see the `ovh-manager-web` associated PR.